docs-ref-services/key-vault.md

```diff
@@ -1,138 +1,107 @@
 
 Use the management library to:
+
 - Create, update, or delete new Key Vault stores
 - Control vault access policies
 - List vaults by subscription or resource group
 - Check for vault name availability
 
-## Install the libraries
-
-### Client library
+Use the client library to:
 
-```bash
-pip install azure-keyvault
-```
+- Access, update, or delete items stored in an Azure Key Vault
+- Get metadata for stored certificates
+- Verify signatures against symmetric keys in Key Vault
 
-## Examples
+Also see [Azure Key Vault samples for Python](/samples/browse/?products=azure-key-vault&languages=python).
 
-The following examples use service principal authentication, which is the recommended sign in method
-for applications that connect to Azure. To learn about service principal authentication, see
-[Authenticate with the Azure SDK for Python](https://docs.microsoft.com/en-us/python/azure/python-sdk-azure-authenticate)
+## Management library
 
-Retrieve the public portion of an asymmetric key from a vault:
+Install the management library with `pip install azure-mgmt-keyvault`, along with the identity library with `pip install azure-identity`.
 
-```python
-from azure.keyvault import KeyVaultClient
-from azure.common.credentials import ServicePrincipalCredentials
+To manage a key vault, an application must first authenticate with the Key Vault service. Authentication is best done using [Azure Managed Identities](/azure/active-directory/managed-identities-azure-resources/overview) (part of Azure Active Directory), and can also also be done with a service principal.
 
-credentials = ServicePrincipalCredentials(
-    client_id = '...',
-    secret = '...',
-    tenant = '...'
-)
+To use Managed Identities, acquire a `DefaultAzureCredential` object as shown in the following code. (To run this code on a developer workstation, see [Using Managed Identities locally](#using-managed-identities-locally)).
 
-client = KeyVaultClient(credentials)
+```python
+# Authentication with Managed Identities
+from azure.identity import DefaultAzureCredential
 
-# VAULT_URL must be in the format 'https://<vaultname>.vault.azure.net'
-# KEY_VERSION is required, and can be obtained with the KeyVaultClient.get_key_versions(self, vault_url, key_name) API
-key_bundle = client.get_key(VAULT_URL, KEY_NAME, KEY_VERSION)
-key = key_bundle.key
+# Acquire default Azure credentials, which are useful in most circumstances.
+credentials = DefaultAzureCredential()
 ```
 
-Retrieve a secret from a vault:
+To use a service principal, acquire a `ServicePrincipalCredential` object as shown in the following code. Review [Authenticate with the Azure SDK for Python](/azure/python/python-sdk-azure-authenticate) for information on creating a service principal and obtaining a tenant ID, a client ID, and application (or client) secret using the `az ad sp create-for-rbac --sdk-auth` command.
```

Comment:
Mixing the authentication libraries here--`ServicePrincipalCredential` is from `msrestazure`, `DefaultAzureCredential` is from `azure-identity`. Track 2 client libraries don't use the former, management libraries don't use the latter.